<launch>
  <!-- Panda -->
  <include file="$(find active_grasp)/launch/panda.launch" />

  <!-- Camera -->
  <include file="$(find realsense2_camera)/launch/rs_camera.launch">
    <arg name="enable_pointcloud" value="true" />
    <arg name="enable_infra1" value="true" />
    <arg name="publish_tf" value="false" />
  </include>

  <!-- AprilTag -->
  <node pkg="apriltag_ros" type="apriltag_ros_continuous_node" name="apriltag_ros" clear_params="true" output="screen">
    <rosparam command="load" file="$(find active_grasp)/cfg/apriltag.yaml" />
    <remap from="image_rect" to="/camera/infra1/image_rect_raw" />
    <remap from="camera_info" to="/camera/infra1/camera_info" />
    <param name="camera_frame" type="str" value="camera_depth_optical_frame" />
    <param name="publish_tag_detections_image" type="bool" value="true" />
  </node>
</launch>
